mobile/assets/i18n/fr-FR.json

```diff
@@ -296,32 +296,32 @@
   "permission_onboarding_permission_limited": "Permission limitée. Pour permettre à Immich de sauvegarder et de gérer l'ensemble de votre bibliothèque, accordez l'autorisation pour les photos et vidéos dans les Paramètres.",
   "permission_onboarding_request": "Immich demande l'autorisation de visionner vos photos et vidéo",
   "profile_drawer_app_logs": "Journaux",
-  "profile_drawer_client_out_of_date_major": "Mobile App is out of date. Please update to the latest major version.",
-  "profile_drawer_client_out_of_date_minor": "Mobile App is out of date. Please update to the latest minor version.",
+  "profile_drawer_client_out_of_date_major": "L'application mobile est obsolète. Veuillez effectuer la mise à jour vers la dernière version majeure.",
+  "profile_drawer_client_out_of_date_minor": "L'application mobile est obsolète. Veuillez effectuer la mise à jour vers la dernière version mineure.",
   "profile_drawer_client_server_up_to_date": "Le client et le serveur sont à jour",
   "profile_drawer_documentation": "Documentation",
   "profile_drawer_github": "GitHub",
-  "profile_drawer_server_out_of_date_major": "Server is out of date. Please update to the latest major version.",
-  "profile_drawer_server_out_of_date_minor": "Server is out of date. Please update to the latest minor version.",
+  "profile_drawer_server_out_of_date_major": "Le serveur est obsolète. Veuillez mettre à jour vers la dernière version majeure..",
+  "profile_drawer_server_out_of_date_minor": "Le serveur est obsolète. Veuillez mettre à jour vers la dernière version majeure.",
   "profile_drawer_settings": "Paramètres",
   "profile_drawer_sign_out": "Se déconnecter",
   "profile_drawer_trash": "Corbeille",
   "recently_added_page_title": "Récemment ajouté",
-  "scaffold_body_error_occurred": "Error occurred",
+  "scaffold_body_error_occurred": "Une erreur s'est produite",
   "search_bar_hint": "Rechercher vos photos",
   "search_page_categories": "Catégories",
   "search_page_favorites": "Favoris",
-  "search_page_motion_photos": "Photos avec mouvement",
+  "search_page_motion_photos": "Photos en mouvement",
```

Comment:
`moving photo` != `motion photo`
Probably `photos animées` is better ?

Reply:
Android parle de photos animées : https://support.google.com/googlecamera/answer/9937175?hl=fr